mlrun/api/utils/alembic.py

```diff
@@ -1,33 +1,42 @@
-import os
 import alembic.config
 import alembic.command
+import os
+import pathlib
+import shutil
 
 from mlrun import mlconf
 
 
 class AlembicUtil(object):
-    class Constants(object):
-        initial_alembic_revision = "11f8dd2dc9fe"
-
     def __init__(self, alembic_config_path):
         self._alembic_config_path = str(alembic_config_path)
         self._alembic_config = alembic.config.Config(self._alembic_config_path)
-        self._current_revision = None
+        self._alembic_output = ""
 
     def init_alembic(self, from_scratch: bool = False):
-        if (
-            not from_scratch
-            and os.path.isfile(self._get_db_file_path())
-            and not self._get_current_revision()
-        ):
+        revision_history = self._get_revision_history()
```

Comment:
It's hard to understand (from both naming + reading the actual code) what's returned here, is this just an ordered list of revision ids ?
consider doing any/all of:
* naming it differently
* add type hints
* add tests (This is a actually general suggestion, this util starting to have gather logic which is not so trivial, I guess adding tests will help someone that is not you touching the code here)